compiler/rustc_mir_transform/src/generator.rs

```diff
@@ -1445,8 +1385,6 @@ pub(crate) fn mir_generator_witnesses<'tcx>(
     tcx: TyCtxt<'tcx>,
     def_id: LocalDefId,
 ) -> Option<GeneratorLayout<'tcx>> {
-    assert!(tcx.sess.opts.unstable_opts.drop_tracking_mir);
-
     let (body, _) = tcx.mir_promoted(def_id);
```

Comment:
> Concretely, does this use analysis MIR before or after drop elaboration for generator auto-traits? If the answer is "after", I'm quite concerned, since we so far specifically avoided making drop elaboration details matter for big compile-time decisions.

Before drop elaboration. The choice was to have "the same MIR as borrowck".